codespell_lib/data/dictionary_usage.txt

```diff
@@ -25,6 +24,7 @@ mohammedans->muslims
 mom-test->user test, acceptance test, validation test, ease-of-use test, friendliness test, useability test,
 muhammadan->muslim
 muhammadans->muslims
+sanity->quick, initial, confidence, coherence, soundness, calibration, rationality,
```

Comment:
I don't understand the purpose of this entry.

Reply:
"Sanity" isn't usually used in code outside of "sanity check", and while we don't have support for two word sequences (https://github.com/codespell-project/codespell/pull/1657), this is the best we can do.

Reply:
Usage of codespell is not necessarily limited to source code. In such cases, as with `white list`, we don't check the isolated first word to avoid false positives.